roles/confluent.variables/defaults/main.yml

```diff
@@ -509,6 +531,9 @@ ksql_ssl_enabled: "{{ssl_enabled}}"
 ### Boolean to enable mTLS Authentication on ksqlDB
 ksql_ssl_mutual_auth_enabled: "{{ ssl_mutual_auth_enabled }}"
 
+### Only use to customize directory ksqlDB writes log files to. Default location is /var/log/confluent/ksql. NOTE- ksql.appender_log_path is deprecated
```

Comment:
Modify this variable to customize  the directory that ksqlDB writes log files to. Default location is /var/log/confluent/ksql. NOTE- ksql.appender_log_path is deprecated.